jsettlers.tools/build.gradle

```diff
@@ -13,6 +13,10 @@ dependencies {
     implementation project(':jsettlers.network')
     implementation project(':jsettlers.testutils')
     implementation project(':jsettlers.tests')
+    // https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-core
```

Comment:
This is really beyond the scope of this pr.
Also I am not going to add log4j or any other big logging framework to jsettlers (except for some very good reason ofc)

Reply:
Removed